src/Raven.Client/Http/RequestExecutor.cs

```diff
@@ -775,28 +775,10 @@ private void ThrowFailedToContactAllNodes<TResult>(RavenCommand<TResult> command
                 message += $"I've tried to access the following nodes: {string.Join(",", _nodeSelector?.Topology.Nodes.Select(x => x.Url) ?? new string[0])}.";
             }
 
-            var firstException = command.FailedNodes.Values.First();
-            
-            // this means that all exception has the same type
-            if (command.FailedNodes.All(x=> ExceptionAreEqual(x.Value, firstException)))
-            {
-                throw new AllTopologyNodesDownException(message, _nodeSelector?.Topology, firstException);
-            }
-
             throw new AllTopologyNodesDownException(message, _nodeSelector?.Topology,
```

Comment:
Here, I want the message to also include:

[ URL Used -> Error message ], [ URL Used 2 -> Error message ], [ URL Used  3-> Error message ],

Reply:
Done